daemon/stop.go

```diff
@@ -22,8 +22,8 @@ func (daemon *Daemon) ContainerStop(name string, seconds *int) error {
 		return err
 	}
 	if !container.IsRunning() {
-		err := fmt.Errorf("Container %s is already stopped", name)
-		return errors.NewErrorWithStatusCode(err, http.StatusNotModified)
+		nil := fmt.Errorf("Container %s is not running", name)
```

Comment:
Thanks @dattatrayakumbhar04 for your contribution. However I am afraid that this change is not what we want. Maybe the following is better:

original code:
```
		err := fmt.Errorf("Container %s is already stopped", name)
		return errors.NewErrorWithStatusCode(err, http.StatusNotModified)
```

We could change two lines into single one line:
```
return errors.NewErrorWithStatusCode(nil, http.StatusNotModified)
```